db/db_compaction_test.cc

```diff
@@ -3333,10 +3336,14 @@ TEST_F(DBCompactionTest, SuggestCompactRangeNoTwoLevel0Compactions) {
 
 INSTANTIATE_TEST_CASE_P(DBCompactionWaitForCompactTest,
                         DBCompactionWaitForCompactTest,
-                        ::testing::Values(std::make_tuple(false, false),
-                                          std::make_tuple(false, true),
-                                          std::make_tuple(true, false),
-                                          std::make_tuple(true, true)));
+                        ::testing::Values(std::make_tuple(false, false, false),
```

Comment:
wondering if there's better way to generate this permutation

Reply:
Should be `::testing::Combine(testing::Bool(), testing::Bool(), testing::Bool())`